src/js/popup/popup-new.js

```diff
@@ -137,7 +229,25 @@
           });
           sendRelayEvent("Panel", "click", userIconPreference);
           return stylePrefToggle(userIconPreference);
-        });        
+        });
+      },
+      isSortaAURL: (str) => {
```

Comment:
I still love this function name :laughing:

Reply:
Same. When I was porting this over from @codemist - I was very much _"No notes. A+"_